.nvmrc

```diff
@@ -0,0 +1 @@
+6.10.0
```

Comment:
Not sure what you mean by it. `travis-build` does not. Users do, via the content of `.travis.yml`. This will be passed to these processes as `config[:node_js]`